src/occupational_classification/_config/sic_soc_llm_config.toml

```diff
@@ -0,0 +1,20 @@
+[sic_soc_data]
```

Comment:
Can you rename the name of the config to something more specific, like sa_soc_config.toml?